docs/contents/components/forms/checkbox/accessibility.mdx

```diff
@@ -8,22 +8,50 @@ tab: Accessibility
 
 The `Checkbox` follows the[WAI-ARIA - Checkbox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/checkbox) for accessibility.
 
-The following example is valid for grouping using `Fieldset`.
+If not using `CheckboxGroup`, use `Fieldset` to group checkboxes.
 
 ```tsx live=false
-<Fieldset>
-  <Legend color="primary">好きなキャラクターを全て選んでください。</Legend>
-  <Checkbox value="孫悟空">孫悟空</Checkbox>
-  <Checkbox value="ベジータ">ベジータ</Checkbox>
-  <Checkbox value="フリーザ">フリーザ</Checkbox>
-</Fieldset>
+const [values, setValues] = useState([false, false])
+
+const allChecked = values.every(Boolean)
+const isIndeterminate = values.some(Boolean) && !allChecked
+
+return (
+  <Fieldset legend="キャラクター">
+    <VStack gap="sm">
+      <Checkbox
+        isChecked={allChecked}
+        isIndeterminate={isIndeterminate}
+        onChange={(e) => setValues([e.target.checked, e.target.checked])}
+      >
+        地球人
+      </Checkbox>
+
+      <VStack pl="md" gap="sm">
+        <Checkbox
+          isChecked={values[0]}
+          onChange={(e) => setValues([e.target.checked, values[1]])}
+        >
+          孫悟空
+        </Checkbox>
+
+        <Checkbox
+          isChecked={values[1]}
+          onChange={(e) => setValues([values[0], e.target.checked])}
+        >
+          孫悟飯
+        </Checkbox>
+      </VStack>
+    </VStack>
+  </Fieldset>
+)
 ```
 
 If not using FormControl, set aria-labelledby on the CheckboxGroup.
```

Comment:
Please fix as following.

If not using `FormControl`, set `aria-labelledby` on the `CheckboxGroup`.

Reply:
I mean to add Backquote.

Reply:
I have made the corrections. Please check it.